lib/shopify-cli/tunnel.rb

```diff
@@ -82,6 +86,34 @@ def auth(ctx, token)
       ctx.system(File.join(ShopifyCli::ROOT, 'ngrok'), 'authtoken', token)
     end
 
+    ##
+    # will return the statistics of the current running tunnels
+    #
+    # #### Returns
+    #
+    # * `stats` - the hash of running statistics returning from the ngrok api
+    #
+    def stats
+      response = Net::HTTP.get_response(NGROK_TUNNELS_URI)
+      JSON.parse(response.body)
+    rescue
```

Comment:
What exception are we anticipating here?

Reply:
This being called without the Tunnel running or possible invalid format from the endpoint.
Pretty much anytime you are calling an endpoint, in my opinion, you cannot fully trust the response.